01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,9 @@
+const personalInformationObj = require("./04-names");
```

Comment:
### Informational (No Change Needed)
[Dependency Management]
Its often a best practice to import only the things you need from a module, instead of importing the entire module as you've done here.
You can do this by the following code:
```js
const { name, lastName} = require("./04-names.js");
```

There are a few reasons this can be preferrable, one being that you are being much more explicit up front about what your code needs and what it doesn't. Here, if someone added 10 names to `./04-names.js` you'd implicitly be loading all of them (the node runtime might actually see you didn't use them and optimize that away through something called tree shaking, but I don't believe it does last I checked). By specifically calling out the 2 you need, its more clear what your code depends on. 

On the flip side if you import many, many dependencies from the same module this may become onerous - at which point the syntax you used may be better for readability and understandability.